public/app/features/plugins/extensions/utils.tsx

```diff
@@ -291,10 +292,6 @@ export function readOnlyCopy<T>(value: T, _log: ExtensionsLog = log): T {
     return value;
   }
 
-  if (config.featureToggles.extensionsReadOnlyProxy) {
-    return getReadOnlyProxy(value);
-  }
-
```

Comment:
Removing this feature toggle as I don't think we could ever activate it without removing all RTK related code